src/app.py

```diff
@@ -295,18 +343,17 @@ def plot_altair(
     vote_count = str(round(filtered_movies["vote_count"].mean()))
 
     # Genre graphs
-
     
     studios_list = []
     studios_str = ""
 
     if revenue_selected is not None:
-        studios_list += [point["y"] for point in revenue_selected["points"]]
-        studios_str = "for the Studios: "
+        studios_list = [point["y"] for point in revenue_selected["points"]]
+        studios_str = "for "
         studios_str += ", ".join(studios_list)
     if vote_selected is not None:
-        studios_list += [point["y"] for point in vote_selected["points"]]
-        studios_str = "for the Studios: "
+        studios_list = [point["y"] for point in vote_selected["points"]]
```

Comment:
I think this will cause a bug:
1. Click on a studio boxplot in revenue plot
    - filtering works as expected
2. Click on a studio boxplot in vote plot
    - filtering works as expected
3. Click again on a studio boxplot in the revenue plot
    - filtering stops working
This happens because when we get the callback from the revenue-plot click in step 3, `vote_selected` is not `None`. So it overwrites `stusios_list` with only the selection in `vote_selected`

Reply:
Thank you for help spotting this, I will revert the change. :)